docs/quickstart.md

```diff
@@ -0,0 +1,84 @@
+## Deploy the application to a cluster
+
+### Prerequisites
+
+- [gcloud CLI](https://cloud.google.com/sdk/docs/install)
+- [kubectl CLI](https://kubernetes.io/docs/tasks/tools/)
+- [skaffold CLI](https://skaffold.dev/docs/install/)
+
+---
+
+### Steps
+
+1. **[Create a Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project)**
+   or use an existing project. Set the `PROJECT_ID` environment variable and
+   ensure the Google Kubernetes Engine and Cloud Monitoring APIs are enabled.
+
+    ```bash
+    export PROJECT_ID="<your-project-id>"
+    gcloud services enable \
+      container.googleapis.com \
+      monitoring.googleapis.com \
+      --project ${PROJECT_ID}
+    ```
+2. **Clone this repository.**
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/point-of-sale.git
+    cd point-of-sale
```

Comment:
Are we intentionally referencing the "old" directory path, in this tutorial?

Reply:
This is pointing to the new directory path. So we are cloning the repo and moving into the repo.
We are not moving into the `point-of-sale-app` folder inside the repo which is the old path